.storybook/preview-head.html

```diff
@@ -10,4 +10,8 @@
   function __(str) {
     return str;
   }
+  
+  window.jest = {
```

Comment:
1. why do we need storybook code to be aware of jest? vs a mock in a test?
2. this is pure JS (doesnt go via webpack) hence you might shouldnt use ES6.

Reply:
because i'm using fixtures file in order to be more dry.
https://github.com/theforeman/foreman/pull/5188/files#diff-7f590775f94b18cdd0bc39583a9a7d40R6